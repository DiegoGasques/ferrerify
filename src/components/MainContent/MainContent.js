import React from "react";

export const MainContent = ({ children }) => {
  return <div className="MainContent">{children}</div>;
};

export default MainContent;
