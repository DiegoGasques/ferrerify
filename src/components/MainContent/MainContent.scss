.MainContent {
  padding: 1.5rem;
}
