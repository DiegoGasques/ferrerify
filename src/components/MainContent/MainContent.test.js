import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { MainContent } from "./MainContent";

describe("[COMPONENT] MainContent", () => {
  it("Tests for changes in the rendered component", () => {
    const wrapper = shallow(<MainContent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
