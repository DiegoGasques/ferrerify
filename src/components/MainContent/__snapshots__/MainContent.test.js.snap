// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] MainContent Tests for changes in the rendered component 1`] = `
<div
  className="MainContent"
/>
`;
